### PR TITLE
Update docstring for `rclcpp::Node::now()` and `rclcpp::LifecycleNode::now()`

### LINKED ISSUE #2688: Minor suggestion for `rclcpp::Node::now()` documentation string
https://github.com/ros2/rclcpp/blob/e64627004f4265560addf3532510a83602be5aef/rclcpp/include/rclcpp/node.hpp#L1433-L1439

Notice how "clock_type" is never referenced in the `node.hpp` file.
I believe this is a copy-paste from `rclcpp::Clock::now()` method docstring.

Maybe it's better to say something like:
```
Returns current time from the node time source.
If `use_sim_time` parameter is set to `true` while
constructing the `Node`, the simulation time is returned,
otherwise system time is returned.
```

### PR DESCRIPTION
**Pull request by roncapat**
Closes #2688

CC: @fujitatomoya

### CODE REVIEW

File: rclcpp/include/rclcpp/node.hpp [hunk 1]
```diff
@@ -1430,8 +1430,11 @@ class Node : public std::enable_shared_from_this<Node>
   rclcpp::Clock::ConstSharedPtr
   get_clock() const;
 
-  /// Returns current time from the time source specified by clock_type.
+  /// Returns current time from the node time source.
```
Comment (fujitatomoya) [inline rclcpp/include/rclcpp/node.hpp]: i agree to remove `clock_type` from here, but technically it is not always come from time source.

```suggestion
  /// Returns current time from the node clock.
```

File: rclcpp/include/rclcpp/node.hpp [hunk 2]
```diff
@@ -1430,8 +1430,11 @@ class Node : public std::enable_shared_from_this<Node>
...
   /**
+   * If `use_sim_time` parameter is set to `true` the simulation time is returned,
+   * otherwise system time is returned.
+   *
```
Comment (fujitatomoya) [inline rclcpp/include/rclcpp/node.hpp]: i do not think this is the correct description, if the Node is constructed with `options.clock_type(RCL_SYSTEM_TIME)` or `options.clock_type(RCL_STEADY_TIME)`. i would remove this description.

```suggestion
```

**Review by fujitatomoya — CHANGES_REQUESTED** [review]

**Review by fujitatomoya — APPROVED** [review]
lgtm.

@roncapat DCO missing, can you fix that?

**Comment by roncapat** [comment]
Done :)

**Pull request merged**